WhatsInSeason/R/in_season_test.R

```diff
@@ -0,0 +1,14 @@
+## A basic prototype to test what the WhatsInSeason package will do.
```

Comment:
Nice example :)

Reply:
Great idea to already come up with this example!